torch/_inductor/fx_passes/pre_grad.py

```diff
@@ -458,3 +493,69 @@ def transpose_matmul(A: torch.Tensor, B: torch.Tensor, Atrans: bool, Btrans: boo
     if Btrans:
         B = B.transpose(-1, -2)
     return torch.matmul(A, B)
+
+
+_pointwise_ops = [
+    operator.add,
+    operator.sub,
+    operator.mul,
+    operator.iadd,
+    operator.isub,
+    operator.imul,
+    operator.itruediv,
+    operator.ifloordiv,
+]
+
+
+@register_graph_pattern(
```

Comment:
I'm not familiar with this function; does it ensure no other uses?

Reply:
Yes, the ```CallFunction``` pattern should match exact the same number of users.